docs/graphviz.rst

```diff
@@ -123,10 +123,60 @@ example above.
 The reStructuredText will be parsed, and after resolving any cross references,
 will be substituted back into the graph definition as follows:
 
-- A :graphvizattr:`label` will be generated from the text content.
-- If there is at least one ``reference`` node, only the last such node is
+- A :graphvizattr:`label` will be generated from the cross reference's text content.
+- If there is at least one cross referable target, only the last such target is
```

Comment:
If the parsed fragment contains at least one hyperlink, only the last such hyperlink is considered, and: